docs/source/configuration.rst

```diff
@@ -374,6 +374,19 @@ to use them for templated. In the above example, you might define a file at
         return "GROUP BY 1,2"
 
 
+If you need root-level functions, you can define them in __init__.py.
```

Comment:
```suggestion
If you need library root-level functions, you can define them in the library's __init__.py.
```
Again worth being specific here as these terms are common in the scope of a larger project that could have multiple packages/__init__s and as shown in the test case the function might be root-level for the library but not necessarily in the root directory of the repository.